.kokoro/docker/docs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ubuntu:22.04
+from ubuntu:24.04
```

Comment:
Looks like `E: Package 'python3-distutils' has no installation candidate` does not work on ubuntu 24.04